src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java

```diff
@@ -0,0 +1,5 @@
+package com.agilethought.atsceapi.exception;
+
+public class ErrorMessage {
+    public static final String badRequestMessage = "You cannot obtain the ";
```

Comment:
"You cannot obtain the " I don't thinks this is a good messages for `BadRequests messages`. @EmiliaGuillen-dev and @luisgalindoluna what do you think?